roles/openshift_expand_partition/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 - name: Ensure growpart is installed
   yum: pkg=cloud-utils-growpart state=present
+  when: not is_containerized | bool
+
+- name: Determine if growpart is installed
+  command: "rpm -q cloud-utils-growpart"
+  register: has_growpart
+  failed_when: "has_growpart.cr != 0 and 'package cloud-utils-growpart is not installed' not in has_growpart.stdout"
+  changed_when: false
+  when: is_containerized | bool
```

Comment:
@twiest @wshearn Are you guys actually using this?

If so, I would like to suggest a rename from openshift_expand_partition to something more generic, since it isn't doing any openshift related configuration.

If not, I would like to propose we remove it in favor of leveraging cloud-init to handle growing/resizing the partition directly.